docs/source/configuration.rst

```diff
@@ -88,6 +109,83 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the :code:`.sqlfluff` config files.
+
+Common rule configurations can be set in the :code:`[sqlfluff:rules]` section.
+
+For example:
+
+.. code-block:: cfg
+
+   [sqlfluff:rules]
+   tab_space_size = 4
+   max_line_length = 80
+   indent_unit = space
+   comma_style = trailing
+   allow_scalar = True
+   single_table_references = consistent
+   unquoted_identifiers_policy = all
+
+Rule specific configurations are set in rule specific subsections.
+
+For example, enforce that keywords are upper case by configuring the rule
+:class:`L010 <sqlfluff.core.rules.Rule_L010>`:
+
+.. code-block:: cfg
+
+    [sqlfluff:rules:L010]  # Keywords
+    capitalisation_policy = upper
+
+All possible options for rule sections are documented in :ref:`ruleref`.
+
+For an overview of the most common rule configurations that you may want to
+tweak, see `Default Configuration`_ (and use :ref:`ruleref` to find the
+available alternatives).
+
+Enabling and Disabling Rules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To disable individual rules, set :code:`exclude_rules` in the top level section
+of sqlfluff configuration. The value is a comma separated list of rule ids.
+
+For example, to disable the rules :class:`L022 <sqlfluff.core.rules.Rule_L022>`
+and :class:`L027 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    exclude_rules = L022, L027
+
+To enable individual rules, configure :code:`rules`, respectively.
+
+For example, to enable :class:`L027 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    rules = L027
+
+If both :code:`exclude_rules` and :code:`rules` have non-empty value, then the
+excluded rules are removed from the rules list. This allows for example
+enabling common rules on top level but excluding some on subdirectory level.
+
+Additionally, some rules have a special :code:`force_enable` configuration
+option, which allows to enable the given rule even for dialects where it is
+disabled by default. The rules that support this can be found in the
+:ref:`ruleref`.
+
+The default values can be seen in `Default Configuration`_.
+
+See also: `Ignoring Errors & Files`_.
+
+..
+    TODO ^ There is also :ref:`inline_ignoring_errors` with overlapping
+    content. Might be worth reviewing to ensure they are both saying the same
+    thing and sync up if not.
```

Comment:
I thought I'd leave this as a hidden comment:

<img width="719" alt="Näyttökuva 2022-1-19 kello 0 09 06" src="https://user-images.githubusercontent.com/4446608/150026657-2d52f2d2-add8-427d-b003-0c1041c77905.png">

Or it can be removed & create an issue about it?

Reply:
That looks good to me. I say include it.